packages/datadog-plugin-restify/test/index.spec.js

```diff
@@ -174,6 +183,43 @@ describe('Plugin', () => {
             })
           })
         })
+
+        it('should allow handleUncaughtException', done => {
+          const server = restify.createServer({
+            handleUncaughtExceptions: true,
+            log: {
+              trace: function () {},
+              warn: function () {}
```

Comment:
we are `log.warn`'ed at startup about domains being a deprecated feature in Node.js & restify, but there's still no replacement in Node.js for this capability. providing log here keeps this deprecation notice from printing.
 
if we provide log, we also have to provide log.trace. currently no other trace levels appear to be hit. we could also provide `info` `error` and `debug` levels as well, for forward compatibility, but they do not seem to matter right now.

Reply:
I'd say it's fine if there is additional output, no need to hide it (unless it's extremely verbose).